src/style/designTokens.ts

```diff
@@ -0,0 +1,73 @@
+import {colorsPalette} from './colorsPalette';
+
+export const designTokens = {
+  ...colorsPalette,
```

Comment:
I wouldn't include the system colors palette in the tokens, It might be good for us to have a clear separation between the two objects in the future. 
In the Colors class you can load the all together